packages/expect/src/__tests__/assertion_counts.test.js

```diff
@@ -30,6 +32,7 @@ describe('.assertions()', () => {
 });
 
 describe('.hasAssertions()', () => {
+  skipOnWindows.suite();
```

Comment:
what happened with this test? was there a "newline" issue?

Reply:
This test failed on window build into the 'continuous-integration/appveyor/pr — AppVeyor build failed' task without any reason about the code changed in my PR. So I used 'skipOnWindows.suite()' as many other cases where is used this approach